README.md

```diff
@@ -2,7 +2,7 @@
   <img src="docs/tersets.png" alt="TerseTS", width="400">
 </h1>
 
-TerseTS is a library that provides methods for lossless and lossy compressing time series. To match existing literature the methods are organized based on [Time Series Compression Survey](https://dl.acm.org/doi/10.1145/3560814). The library is implemented in Zig and provides a Zig-API and C-API with [bindings](#usage) for other languages.
+TerseTS is a library that provides methods for lossless and lossy compressing time series. To match existing literature the lossy compression methods are organized in a [hierarchy](docs/hierarchy.pdf) based on [Time Series Compression Survey](https://dl.acm.org/doi/10.1145/3560814).The library is implemented in Zig and provides a Zig-API and C-API with [bindings](#usage) for other languages.
 
 # Getting Started
```

Comment:
Maybe remove "# Getting Started" and make all titles in the README start with "#" instead of a combination of "#" and "##". Since the README now has folds that hide most of the instructions, having sections and subsections instead of just sub-sections seems like overkill, and I get confused by the "Getting Started" section every time I see it since it has no content.